src/main/java/org/team1540/robot2024/subsystems/shooter/Shooter.java

```diff
@@ -79,6 +81,11 @@ public void periodic() {
         Logger.processInputs("Shooter/Flywheels", flywheelInputs);
         MechanismVisualiser.setShooterPivotRotation(getPivotPosition());
 
+        if (RobotState.isDisabled()){
+            stopFlywheels();
+            stopPivot();
```

Comment:
This is already happening in several other places, such as the drivetrain, and we'd like to still be safe if some command does not terminate properly. I don't think we want a mechanism to move instantly as we enable just because someone messed up writing a command.

Reply:
I mean yes, but what this means is that we won't know that we have a problem because we won't find it as clearly in testing